producer/index.js

```diff
@@ -9,16 +10,56 @@ const addresses = ['2MwcVyhWzUrS6AK8mwAZPjBfeQqtdEGgeSg',
 ];
 
 const bitcoinSocket = new btSocket({network: 3});
+const blockExplorer = btExplorer.usingNetwork(3);
 
-const context = rabbitJs.createContext('amqp://user:pass@rabbit/vhost');
+const rabbitHost = process.env.RABBIT_HOST || 'localhost';
+
+const queueName = 'transaction';
+
+function intersection(arr1, arr2) {
+    return arr1.filter((i) => arr2.indexOf(i) !== -1);
+}
+
+function filterTransactions(transactionLog) {
+    const fundingTransactions = transactionLog.txs.filter((tx) => {
+        const ins = tx.inputs.map((item) => item.prev_out.addr);
+        const outs = tx.out.map((item) => item.addr);
+        const isFundingTransaction =
+            !intersection(ins, addresses).length && intersection(outs, addresses).length;
+        return isFundingTransaction;
+    });
+    return fundingTransactions;
+}
+
+async function sendOldTransactions(callback) {
+    const limit = 100;
```

Comment:
well done with this one. I figured out the `getMultiAddress` limitation with 100 txs 👍